cpp/ql/src/Metrics/Internal/IncludeResolutionStatus.ql

```diff
@@ -0,0 +1,19 @@
+/**
+ * @name Count of includes that failed to be resolved as well as compilation error due to undefined token
+ * @description A count of all failed includes and compilation errors due to undefined tokens.
+ * @kind table
+ * @tags summary telemetry include resolution
+ * @id cpp/telemetry/incliude-resolution-status
```

Comment:
I would follow the naming of other queries in this directory
```suggestion
 * @id cpp/include-resolution-status
```

Reply:
Fixed